src/test/groovy/org/cassalog/core/CassalogTest.groovy

```diff
@@ -104,6 +105,28 @@ class CassalogTest extends CassalogBaseTest {
     assertChangeSetEquals(rows[1], new ChangeSet(version: change2Id, author: 'admin', description: 'test'))
   }
 
+  @Test
```

Comment:
Just added one more commit to ensure that the keyspaces created on the test are deleted after the test. Let me know if you do not like it and I can remove it.